dll/win32/shell32/lang/id-ID.rc

```diff
@@ -924,8 +924,8 @@ BEGIN
     IDS_CANTDISCONNECT "Tidak bisa memutuskan (Kode Kesalahan: %lu)."
     IDS_NONE "(Tidak ada)"
 
-    IDS_EXPAND "Exp&and"
-    IDS_COLLAPSE "Coll&apse"
+    IDS_EXPAND "Perlu&as"
+    IDS_COLLAPSE "Per&sempit"
```

Comment:
Note that for these two strings you can use the same accelerator (I don't know if e.g. "P" could be used if it doesn't collide with anything else) because either one of these strings is shown at any time.

Reply:
review addressed. thanks